src/cookies.ts

```diff
@@ -0,0 +1,38 @@
+export function get(key: string): string | null {
```

Comment:
I didn't end up using it but removing this module seemed like a waste - might come handy in the future.